backend/zeno_backend/database/select.py

```diff
@@ -2660,3 +2660,48 @@ async def system_exists(project_uuid: str, system_name: str) -> bool:
         return bool(exists[0][0])
     else:
         raise HTTPException(status_code=500, detail="Could not check if system exists")
+
+
+async def chart_config(project_uuid: str) -> ChartConfig:
+    """Get a project's chart configuration.
+
+    Args:
+        project_uuid (str): the project for which to fetch the config.
+
+    Returns:
+        ChartConfig | None: the config if there is one, otherwise None.
+    """
+    async with db_pool.connection() as conn:
+        async with conn.cursor() as cur:
+            await cur.execute(
+                "SELECT config from chart_config WHERE project_uuid = %s;",
+                [project_uuid],
+            )
+            config = await cur.fetchall()
+
+    if len(config) == 0:
+        return ChartConfig(project_uuid=project_uuid)
+    return config[0][0]
```

Comment:
Do we need to cast or anything here? How does Python know this is a ChartConfig? idk types in python are confusing to me

Reply:
No, don't need to cast.